src/accelerate/utils/environment.py

```diff
@@ -61,17 +62,44 @@ def are_libraries_initialized(*library_names: str) -> Dict[str, bool]:
     return [lib_name for lib_name in library_names if lib_name in sys.modules]
 
 
+def get_gpu_info():
+    """
+    Gets GPU count and names using `nvidia-smi` instead of torch to not initialize CUDA.
+
+    Largely based on the `gputil` library.
+    """
+    if platform.system() == "Windows":
+        # If platform is Windows and nvidia-smi can't be found in path
+        # try from systemd rive with default installation path
+        command = spawn.find_executable("nvidia-smi")
+        if command is None:
+            command = "%s\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe" % os.environ["systemdrive"]
+    else:
+        command = "nvidia-smi"
+    # Returns as list of `n` GPUs and their names
+    output = subprocess.check_output(
+        [command, "--query-gpu=count,name", "--format=csv,noheader"], universal_newlines=True
+    )
+    gpus = output.split(os.linesep)
+    # Get names from output
+    gpu_count = len(gpus)
+    gpu_names = [gpu.split(",")[1].strip() for gpu in gpus]
```

Comment:
When running locally, this line failed because `output` had a trailing `\n`, resulting in `gpus == ['1, NVIDIA GeForce RTX 2060', '']`. Maybe strip the output before splitting?

Reply:
Done!